app/src/main/java/com/sun/hero_01/data/source/remote/fetchdata/ParseJson.kt

```diff
@@ -0,0 +1,101 @@
+package com.sun.hero_01.data.source.remote.fetchdata
+
+import com.sun.hero_01.data.model.*
+import com.sun.hero_01.utils.KeyEntityType
+import org.json.JSONObject
+
+class ParseJson {
+
+    fun heroDetailParseJson(jsonObject: JSONObject) = jsonObject.run {
```

Comment:
tương tự đặt lại tên các fun ở dưới e
```suggestion
    fun onParseJsonToHero(jsonObject: JSONObject) = jsonObject.run {
```